curriculum/challenges/english/02-javascript-algorithms-and-data-structures/object-oriented-programming/understand-own-properties.english.md

```diff
@@ -27,7 +27,9 @@ tests:
   - text: <code>ownProps</code> should include the values <code>"numLegs"</code> and <code>"name"</code>.
     testString: assert(ownProps.indexOf('name') !== -1 && ownProps.indexOf('numLegs') !== -1, '<code>ownProps</code> should include the values <code>"numLegs"</code> and <code>"name"</code>.');
   - text: Solve this challenge without using the built in method <code>Object.keys()</code>.
-    testString: assert(!/\Object.keys/.test(code), 'Solve this challenge without using the built in method <code>Object.keys()</code>.');
+    testString: assert(!/Object(\.(keys)|\[('|"|\`).+\2\3\])/.test(code), 'Solve this challenge without using the built in method <code>Object.keys()</code>.');
```

Comment:
Seems like the .+ would capture too much, but it doesn't really matter.

Reply:
What do you recommend instead?

Reply:
Why is it even needed?

Reply:
I think it was left over from something else I tried before.  I have removed it and actually refactored the regex.  Also, I went ahead and remove the unnecessary assert message arguments to clean it up a bit more.

Let me know if you see any other problems.